bundles/debian/Containerfile

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 The cert-manager Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM docker.io/library/debian:11-slim as debbase
```

Comment:
Do we want to be using a hash as the tag for this? We could have a cronjob action which checks if there is a newer one and creates a PR if there is.

Reply:
I think generally, in most situations I would favour using a digest rather than a tag, but in this case I'm explicitly looking to always use the latest available.

If Debian release an update to the `ca-certificates` package we need to publish an updated image ASAP. In the worst case - e.g. if a CA private key was published online - then our updated image could be one of the most important security updates ever for users who rely on us.

If we have a PR to review for updating base images, that means we can only really rely on the update being made during UK/European office hours / waking hours. If we're all asleep there won't be an update until one of us wakes up and notices.

We could automatically merge the "bump" PRs but that still adds several failure cases (e.g. the github API gives a 500 when creating the PR, the automatic merge bot fails, etc)

The intention of this PR is to absolutely minimise the delay between 1) an update being available in Debian and 2) our updated image being published. In my own tests, I had the GCB job triggered every 30 minutes but I think we could be faster than that!